Tests/MessagingInApp/Gist/Network/SSE/SseRetryHelperTest.swift

```diff
@@ -0,0 +1,309 @@
+@testable import CioInternalCommon
+@testable import CioMessagingInApp
+import SharedTests
+import XCTest
+
+/// Tests for `SseRetryHelper` actor.
+class SseRetryHelperTest: XCTestCase {
+    private var loggerMock: LoggerMock!
+    private let testGeneration: UInt64 = 1
+
+    override func setUp() {
+        super.setUp()
+        loggerMock = LoggerMock()
+    }
+
+    // MARK: - Constants
+
+    func test_constants_expectCorrectValues() {
+        XCTAssertEqual(SseRetryHelper.maxRetryCount, 3)
+        XCTAssertEqual(SseRetryHelper.retryDelaySeconds, 5.0)
```

Comment:
These tests wait up to 6 seconds because retryDelaySeconds is 5s? think it cause cause flakiness. Should we  Inject a `clock/sleeper` into SseRetryHelper (or make delay configurable for tests), so unit tests can run in faster (milliseconds)?